Foundation/NSAttributedString.swift

```diff
@@ -7,9 +7,14 @@
 // See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
 //
 
+import CoreFoundation
 
 public class NSAttributedString : NSObject, NSCopying, NSMutableCopying, NSSecureCoding {
     
+    private let _cfinfo = _CFInfo(typeID: CFAttributedStringGetTypeID())
+    private let _string: NSString
```

Comment:
I tried to use `private let _string: String` here, but encountered EXC_BAD_ACCESS at CFString function calls.
I didn't figure out why, but found out that with a `NSString`, everything seemed to be fine.